tide/master/MasterApplication.cpp

```diff
@@ -437,19 +439,20 @@ void MasterApplication::_initTouchListener()
             [this, getWallPos](const int id, const QPointF normalizedPos) {
                 _markers->addMarker(id, getWallPos(normalizedPos));
 #if TIDE_ENABLE_PLANAR_CONTROLLER
-                if (_planarController)
+                if (!_planarController ||
+                    _planarController->getState() != ScreenState::OFF)
+                    return;
+                else
```

Comment:
no else after the return! Which allows to reduce indentation of the following block... + style, when the if is on multiple lines put the return; in {}